Cargo.toml

```diff
@@ -21,8 +21,16 @@ path = "regex_macros/benches/bench_dynamic.rs"
 test = false
 bench = true
 
+[dependencies.regex-syntax]
+path = "regex_syntax"
+version = "*"
```

Comment:
Yeah unfortunately we can't switch underscore-to-dash after the fact just yet :( 

I would like to enable this kind of feature on crates.io though as I suspect it comes up more often than not!